src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -75,13 +75,30 @@ This feature is not yet available in the Asia Pacific (Hong Kong, ap-east-1) or
 
 </Callout>
 
+<BlockSwitcher>
+<Block name="Systems Manager">
 First, place your database connection information (hostname, username, password, port, and database name) into Systems Manager, each as a `SecureString`.
 
 Go to the Systems Manager console, navigate to Parameter Store, and click "Create Parameter". Create five different SecureStrings: one each for the hostname of your database server, the username and password to connect, the database port, and the database name.
 
 Your Systems Manager configuration should look something like this:
 
 ![Storing Database Credentials in SSM](/images/storing-db-creds-in-ssm.png)
+</Block>
+<Block name="Secrets Manager">
+First, place your database connection information (hostname, username, password, port, and database name) into Secrets Manger.
+
+Go to the Secrets Manager console, navigate to Secrets, and click "Store a new secret". You may create the secret in any manner as long as there are `username` and `password` keys defined.
+
+Optionally, you can decide whether to encrypt the secret using the KMS key that Secrets Manager creates or a customer managed KMS key that you create.
+
+You can also optionally configure a rotation schedule and create a lambda or choose an existing one from your account to rotate the database credentials automatically.
+
+When viewing your new secret's value, it should look something like this:
+
+![Storing Database Credentials in Secrets Manager](/images/storing-db-creds-in-secrets-manager.png)
```

Comment:
Hey @chrisbonifacio, in an effort to make our images more accessible could we get an updated alt text for this? It looks like you're referring to the image with the above sentence and from this alt text, it's not descriptive.

Could you please update the alt text so that it describes what's in the image?

Some resources to help write alt text:

https://webaim.org/techniques/alttext/
https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

Reply:
Hey @chrisbonifacio, could we actually get a more descriptive alt text for this? I still think that if you're trying to reference the image then we should explain what we're actually referring to. In this case, if the image is trying to show the key value pairs you have in your table, then we should mention that in the alt text